.prettierrc.yml

```diff
@@ -5,4 +5,4 @@ printWidth: 88
 proseWrap: always
 semi: true
 trailingComma: "es5"
-xmlWhitespaceSensitivity: "ignore"
+xmlWhitespaceSensitivity: "strict"
```

Comment:
Do this belong to this PR?

Reply:
Since I backported from v14, I wanted to keep the XML files identical with v14.
If I remove this, prettier will reformat the XML and back/forward ports with v14 will become more difficult.

Reply:
But won't this affect to the rest of the repo?

Reply:
No, it will just apply less transformations to XML files.

Reply:
Ok, thanks